content/guides/typescript.md

```diff
@@ -136,3 +149,15 @@ declare module "*.svg" {
 ```
 
 Here we declare a new module for SVGs by specifying any import that ends in `.svg` and defining the module's `content` as `any`. We could be more explicit about it being a url by defining the type as string. The same concept applies to other assets including CSS, SCSS, JSON and more.
+
+
+## Build Performance
+
+W> This may degrade build performance.
```

Comment:
What exactly impacts build performance? Using TypeScript in general?

Maybe be a bit more explicit about what "This" means in this context :)